### PR TITLE
Lobby statpanel moved to Status

### PR DESCRIPTION
**Pull request by Cyberboss**
No reason to have it split when only new_players can see it

### CODE REVIEW

**Comment by praisenarsie** [comment]
thanks man.

**Comment by MrStonedOne** [comment]
can you make it so ghosts in the round during pre-game can see, it would be helpful for admin events.

File: code/modules/mob/dead/new_player/new_player.dm [hunk 1]
```diff
@@ -76,22 +76,22 @@
 /mob/dead/new_player/Stat()
 	..()
 
-	if(statpanel("Lobby"))
-		stat("Game Mode:", (SSticker.hide_mode) ? "Secret" : "[GLOB.master_mode]")
-		stat("Map:", SSmapping.config.map_name)
+	if(statpanel("Status"))
+		stat(null, "Game Mode: [SSticker.hide_mode ? "Secret" : "[GLOB.master_mode]"]")
+		stat(null, "Map: [SSmapping.config.map_name]")
```
Comment (MrStonedOne) [inline code/modules/mob/dead/new_player/new_player.dm]: The map is already displayed in the status panel

**Comment by Cyberboss** [comment]
Yeah I'll move it up to /dead

**Pull request merged**